src/select.jsx

```diff
@@ -197,6 +198,10 @@ var classBase = React.createClass({
     });
   },
   onBlur () {
+    if (this.props.onBlur) {
```

Comment:
We may want to do this in the callback of `setState` so it's guaranteed to not be showing the dropdown. Thoughts?